apps/examples/testcase/le_tc/network/tc_net_api.c

```diff
@@ -0,0 +1,75 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_net_api.c
+/// @brief Test Case Example for api() API
+#include "tc_internal.h"
+#include <net/lwip/api.h>
+#include <net/lwip/ipv4/ip_addr.h>
+#include <net/lwip/err.h>
+/**
+* @statitcase           : tc_net_netconn_sendto_n
+* @brief                :
+* @scenario             :
+* @apicovered           : netconn_sendto
+* @precondition         :
+* @postcondition        :
+*/
+static void tc_net_netconn_sendto_n(void)
+{
+	struct netconn *conn = NULL;
+	struct netbuf *buf = NULL;
+	ip_addr_t *addr = NULL;
+	u16_t port = 0;
+	err_t result;
+
+	result = netconn_sendto(conn, buf, addr, port);
+
+	TC_ASSERT_EQ("api", result, ERR_VAL);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @statitcase       : tc_net_netconn_close_n
+* @brief            :
+* @scenario         :
+* @apicovered       : netconn_close
+* @precondition     :
+* @postcondition    :
+*/
+static void tc_net_netconn_close_n(void)
+{
+	struct netconn *conn = NULL;
+	netconn_close(conn);
+}
```

Comment:
Here is no "TC_SUCCESS_RESULT". If this is an one of TC, you should add it. And is there no failure?